features/text-wrap-stable.yml

```diff
@@ -0,0 +1,7 @@
+name: "text-wrap: stable"
+description: "The `text-wrap: stable` CSS declaration wraps text as normal, but does not change wrapping on earlier lines while the text is edited."
```

Comment:
- "as normal" - would be more correct to use "default" instead of "normal"
- "is edited" -  reads like missing a word, does it mean while the text is *being* edited?